redpanda/templates/post-upgrade.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if (include "redpanda.semver" . | semverCompare ">=22.1.4") }}
```

Comment:
Same comment about semver version as before, I believe this should be `22.1.1`.

Reply:
[519e4f1](https://github.com/redpanda-data/helm-charts/pull/117/commits/519e4f193b60c4e2410f2b1042c3eb2d958f5ea7)